src/current/v23.2/role-based-audit-logging.md

```diff
@@ -49,6 +51,10 @@ With the [`sql.log.user_audit` cluster setting]({% link {{ page.version.version
 
 Once `sql.log.user_audit` is set, the default behavior is for role-based SQL audit logging to take effect immediately within user sessions.
 
+`sql.log.user_audit.reduced_config.enabled`
+
+When enabled, the [`sql.log.user_audit.reduced_config.enabled` cluster setting]({% link {{ page.version.version }}/cluster-settings.md %}#setting-sql-log-user-audit-reduced-config-enabled) computes a "reduced" audit configuration based on the user's current role memberships and the current value for the `sql.log.user_audit` cluster setting. The "reduced" audit configuration is computed at the **first SQL event emitted by the user, after the setting is enabled**. When the cluster setting is enabled, CockroachDB computes the audit configuration once at session start, instead of at each SQL event. However with the setting enabled, changes to the audit configuration (user role memberships or cluster setting configuration) are not reflected within a user session. To reflect the configuration changes in auditing behavior, users need to start a new session.
```

Comment:
First mention of 'role memberships' could be a link to https://www.cockroachlabs.com/docs/stable/security-reference/authorization#roles

First mention of 'user session' could be a link to https://www.cockroachlabs.com/docs/v23.1/show-sessions